src/Infrastructure.EntityFramework/Repositories/OrganizationRepository.cs

```diff
@@ -104,13 +104,26 @@ public async Task<ICollection<Core.Entities.Organization>> SearchUnassignedToPro
 
             if (!string.IsNullOrWhiteSpace(ownerEmail))
             {
-                query = from o in query
-                        join ou in dbContext.OrganizationUsers
-                            on o.Id equals ou.OrganizationId
-                        join u in dbContext.Users
-                            on ou.UserId equals u.Id
-                        where u.Email == ownerEmail && ou.Type == OrganizationUserType.Owner
-                        select o;
+                if (dbContext.Database.IsNpgsql())
```

Comment:
You use `EF.Functions.Like` in the first query in this method without testing for `IsNpgsql`. Do you need this check there as well?

Best I can tell, `ILike` is postgres specific and `Like` is for everyone else?

Reply:
> You use EF.Functions.Like in the first query in this method without testing for IsNpgsql. Do you need this check there as well?

Good question! I've been testing this using the unit test `OrganizationRepositoryTests.SearchUnassignedAsync_Works` and I've concluded that `EF.Functions.Like` only doesn't work on the `Email` column.

> Best I can tell, ILike is postgres specific and Like is for everyone else, is that right?

That is right.